### PR TITLE
Automated Test COMMERCE-6264 | master

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

**Comment by leticiachiu** [comment]
https://issues.liferay.com/browse/COMMERCE-6264; cc: @AlessandroIovane

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by iovaneale** [comment]
Starting review

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceEntry.macro [hunk 1]
```diff
@@ -535,6 +589,46 @@ definition {
 			value1 = "${newEntryShortDescription}");
 	}
 
+	macro editTaxCategory {
+		CommerceNavigator.gotoEntry(entryName = "${keyChannel}");
```
Comment (iovaneale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceEntry.macro:593]: Please move this step outside of the macro. Thanks.

**Review by iovaneale — CHANGES_REQUESTED** [review]

**Comment by leticiachiu** [comment]
@iovaneale Fixed.

**Comment by iovaneale** [comment]
ci:test:commerce-functional

**Comment by iovaneale** [comment]
ci:forward

**Comment by iovaneale** [comment]
Forwarded since the failure are not related to the commits in this PR. The pushed test passed.

**Pull request closed without merging**